src/python/pants/backend/docker/goals/package_image.py

```diff
@@ -170,7 +170,11 @@ def get_build_options(
     field_set: DockerFieldSet,
     global_target_stage_option: str | None,
     target: Target,
+    verbose: bool,
 ) -> Iterator[str]:
+    if not verbose:
```

Comment:
Doesn't this mean that we won't get much debug output below?

Reply:
Hmm.. I think you're onto me here. I thought I'd be clever and leverage a option on the docker client here, but as you point out, this flag is more about whether to log the output as debug or info. Got the same train of thought after going to bed yesterday..